pulp_puppet_plugins/pulp_puppet/plugins/db/models.py

```diff
@@ -68,7 +68,7 @@ class Module(FileContentUnit):
     tag_list = ListField()
 
     # Generated at the file level
-    checksum = StringField(required=True)
```

Comment:
It's not possible to save the unit with ``required=True``, and to do ``import_content`` (and then calculate the checksum) the unit must be saved.